components/navigation/BottomBar.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { View, StyleSheet, Dimensions } from "react-native";
+import { ThemedIconButton } from "@/components/theme/ThemedIconButton";
+import { Colors } from "@/constants/Colors";
+import { useThemeColor } from "@/hooks/useThemeColor";
+
+// Get screen width and height
+const { width, height } = Dimensions.get("window");
+
+interface BottomBarProps {
+  leftIcon?: string;
+  leftAction?: () => void;
+  centerIcon?: string;
+  centerAction?: () => void;
+  rightIcon?: string;
+  rightAction?: () => void;
+  colorType?: keyof typeof Colors.light & keyof typeof Colors.dark;
+}
+
+export function BottomBar({
+  leftIcon,
+  leftAction,
```

Comment:
You could use `leftAction = () => {},` here to have a better line 35. Having a default value seems clearer. It should be adapted to `centerAction` and `rightAction` as well.

Reply:
Absolutely right !